testworld/park.go

```diff
@@ -281,6 +283,16 @@ func (h *host) init() error {
 		if err != nil {
 			return err
 		}
+
+		values := map[string]interface{}{
```

Comment:
@vimukthi-git we update the config and add the keys only for testworld. So we can run tests without having to rerun the host creation.